driver/web/admin_permission_policy.csv

```diff
@@ -15,3 +15,5 @@ p, update_configs_gateway, /gateway/api/admin/configs/*, (GET)|(PUT), Update gat
 p, update_configs_gateway, /gateway/api/admin/configs, (GET)|(POST),
 p, delete_configs_gateway, /gateway/api/admin/configs/*, (GET)|(DELETE), Delete gateway configs
 p, delete_configs_gateway, /gateway/api/admin/configs, (GET),
+
+p, webtools_blacklist, /gateway/api/admin/webtoolsblacklist, (GET)|(POST)|(PUT)|(DELETE), Webtools blacklist actions
```

Comment:
In additon to this permission we need to define one more permission called _all_admin_gateway_ which can be used for all gateway admin APIs. The same approach we follow in the other BBs.